spec/support/feature_helpers.rb

```diff
@@ -0,0 +1,9 @@
+# Helper methods for feature specs
+module FeatureHelpers
+  def sign_in_as(user)
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
```

Comment:
This is awesome.

I'm thinking there might also already be a devise test helper for doing this (I could be wrong). But worth investigating.